+logging/logging.m

```diff
@@ -165,7 +165,12 @@ function delete(self)
       end
     end
 
-    function writeLog(self, level, caller, message)
+    function writeLog(self, level, caller, message, varargin)
+        
+      if nargin > 4
+          message = sprintf(message, varargin{:});
+      end

```

Comment:
This only needs to be done if the log level is high enough, right?

Reply:
Why? I think all log levels can have this feature.
Thank you!

Reply:
Oh I see, thanks for the clarification!

Reply:
It would be great if you could also add a test. Thank you for implementing this!

Reply:
That would surely be the best. We should add proper unit test at some point.